src/style.css

```
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --main-color: #00acdc;
+}
+
+.header--nav {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav--list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+}
+
+.nav--item {
+  display: flex;
+  margin: 0 10px;
+  height: 60px;
+  list-style: none;
+}
+
+.nav--link {
+  display: flex;
+  align-items: center;
+  line-height: 14px;
+  text-decoration: none;
+  color: #000;
+}
+
+.is-active {
+  color: var(--main-color);
+}
```

```suggestion
.nav--link:hover,
.is-active {
  color: var(--main-color);
}
```

Don't repeat yourself